CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- Marsha volumes are used only in trashable environments.
```

Comment:
IMO you don't need to add a dot at the end or each item.

Reply:
:ok_hand: